File: python/1048_aumento_de_salario.py
a = float(input())

if a >= 0 and a < 400.01:
    reajuste = a*0.15
    total = a + reajuste
    print(f'Novo salario: {total:.2f}')
    print(f'Reajuste ganho: {reajuste:.2f}')
    print(f'Em percentual: 15 %')
elif a >= 400.01 and a < 800.01:
    reajuste = a*0.12
    total = a + reajuste
    print(f'Novo salario: {total:.2f}')
    print(f'Reajuste ganho: {reajuste:.2f}')
    print(f'Em percentual: 12 %')
elif a >= 800.01 and a < 1200.01:
    reajuste = a*0.10
    total = a + reajuste
    print(f'Novo salario: {total:.2f}')
    print(f'Reajuste ganho: {reajuste:.2f}')
    print(f'Em percentual: 10 %')
elif a >= 1200.01 and a < 2000.01:
    reajuste = a*0.07
    total = a + reajuste
    print(f'Novo salario: {total:.2f}')
    print(f'Reajuste ganho: {reajuste:.2f}')
    print(f'Em percentual: 7 %')
else:
    reajuste = a*0.04
    total = a + reajuste
    print(f'Novo salario: {total:.2f}')
    print(f'Reajuste ganho: {reajuste:.2f}')
    print(f'Em percentual: 4 %')

File: c/c99/1048_aumento_de_salario.c
#include <stdio.h>

int main() {
    float salario, reajuste, ganho;
    int porcentagem;
    scanf("%f", &salario);
    if (0 <= salario && salario <= 400.00) {
        reajuste = salario * 1.15;
        porcentagem = 15;
        ganho = reajuste - salario;
    } else if (400.01 <= salario && salario <= 800.00) {
        reajuste = salario * 1.12;
        porcentagem = 12;
        ganho = reajuste - salario;
    } else if (800.01 <= salario && salario <= 1200.00) {
        reajuste = salario * 1.10;
        porcentagem = 10;
        ganho = reajuste - salario;
    } else if (1200.01 <= salario && salario <= 2000.00) {
        reajuste = salario * 1.07;
        porcentagem = 7;
        ganho = reajuste - salario;
    } else {
        reajuste = salario * 1.04;
        porcentagem = 4;
        ganho = reajuste - salario;
    }
    printf("Novo salario: %.2f\n", reajuste);
    printf("Reajuste ganho: %.2f\n", ganho);
    printf("Em percentual: %d %%\n", porcentagem);

    return 0;
}
